src/termdict/sstable_termdict/sstable/sstable_index.rs

```diff
@@ -85,8 +85,8 @@ impl SSTableIndexBuilder {
         })
     }
 
-    pub fn serialize(&self, wrt: &mut dyn io::Write) -> io::Result<()> {
-        serde_cbor::ser::to_writer(wrt, &self.index).unwrap();
+    pub fn serialize<W: std::io::Write>(&self, wrt: W) -> io::Result<()> {
+        ciborium::ser::into_writer(&self.index, wrt).unwrap();
```

Comment:
`&dyn io::Write` was fine here I think...

can you fix the horrible unwrap however ?